src/main/java/org/spongepowered/api/item/recipe/crafting/ShapelessCraftingRecipe.java

```diff
@@ -45,37 +45,36 @@ static Builder builder() {
         return Sponge.getRegistry().createBuilder(Builder.class);
     }
 
-
     /**
      * Gets the ingredients for this ShapelessRecipe.
      *
      * @return The ingredients
      */
-    Collection<ItemStack> getIngredients();
+    Collection<ItemStackSnapshot> getIngredients();
```

Comment:
Getting ingredients as snapshots makes sense but I wouldn't require them when adding to the builder. Makes working with the API painful (when it could turn them into snapshots itself).